docs/DeveloperGuide.md

```diff
@@ -58,6 +58,16 @@ the following packages.
 * Task: Contains the classes for the different types of tasks.
 * Ui: In charge of interaction with the user.
 
+####SubjectCommand
+![AddSubjectSequenceDiagram](https://user-images.githubusercontent.com/47527482/97392568-0bf0ab80-191d-11eb-99cf-c98d7863ba91.png)
+![AccessSubjectSequenceDiagram](https://user-images.githubusercontent.com/47527482/97392572-0f843280-191d-11eb-8820-f69bdae48e83.png)
+
+####Diagrams
+![TaskCommandClassDiagram](https://user-images.githubusercontent.com/47527482/97393705-211a0a00-191e-11eb-9546-f6b6b0e47bc4.png)
+![FlashcardCommandClassDiagram](https://user-images.githubusercontent.com/47527482/97393768-24ad9100-191e-11eb-8f35-c7ed86d3dd55.png)
+![SubjectCommandClassDiagram](https://user-images.githubusercontent.com/47527482/97393784-25462780-191e-11eb-87b8-1dd4f776f2bf.png)
```

Comment:
![image](https://user-images.githubusercontent.com/15065550/97545275-b8a65800-1a05-11eb-8205-22f4ca52a0b9.png)

I think the rightmost part is cut off.